packages/devtools_app/lib/src/performance/performance_screen.dart

```diff
@@ -375,6 +380,206 @@ class _SecondaryControls extends StatelessWidget {
   }
 }
 
+class EnhanceTracingButton extends StatefulWidget {
+  const EnhanceTracingButton({Key key}) : super(key: key);
+
+  @override
+  State<EnhanceTracingButton> createState() => _EnhanceTracingButtonState();
+}
+
+class _EnhanceTracingButtonState extends State<EnhanceTracingButton>
+    with AutoDisposeMixin {
+  static const _hoverYOffset = 10.0;
+
+  final _tracingServiceExtensions = [
+    profileWidgetBuilds,
+    profileRenderObjectLayouts,
+    profileRenderObjectPaints,
```

Comment:
this list occurs twice. would be good to avoid that to prevent the UI from getting out of sync. I would drive the list of actual buttons from this list of extensions maybe adding some extra fields to the extension object data classes as needed.